src/uwtools/tests/config/test_tools.py

```diff
@@ -374,6 +364,80 @@ def test_realize_config_simple_yaml(tmp_path):
     help_realize_config_simple("simple2.yaml", FORMAT.yaml, tmp_path)
 
 
+def test_realize_config_single_dereference(caplog, tmp_path):
+    path = tmp_path / "a.yaml"
+    supplemental_path = tmp_path / "b.yaml"
+    with writable(path) as f:
+        yaml.dump({"1": "a", "2": "{{ deref }}", "3": "{{ temporalis }}"}, f)
+    with writable(supplemental_path) as f2:
+        yaml.dump({"2": "b", "temporalis": "c"}, f2)
+    tools.realize_config(
+        input_config=path,
+        input_format=FORMAT.yaml,
+        output_file=None,
+        output_format=FORMAT.yaml,
+        supplemental_configs=[supplemental_path],
+        dry_run=True,
+    )
+    expected = """'1': a
+'2': b
+'3': c
+temporalis: c
+"""
+    actual = "\n".join(record.message for record in caplog.records)
+    assert actual == expected
```

Comment:
Nice test. To drive the intent home, we might add a `"deref": "d"` key to `b.yaml` to make it clear that `{{ deref }}` never actually gets rendered, but rather just gets clobbered by `"2" : "b"`.

Reply:
I like it. Added.